experimental/dds/tree/src/Common.ts

```diff
@@ -185,6 +187,7 @@ export function assert(condition: unknown, message?: string, containsPII = false
  */
 export function fail(message: string = defaultFailMessage, containsPII = false): never {
 	if (process.env.NODE_ENV !== 'production') {
```

Comment:
What? Surely this returns true in the browser and hits the debugger line...?

Reply:
Just throws, because process doesn't exist so you get a null deref.  Looked pretty strange to me too, but in cases where I had any doubts I opted to disable the rule and move on rather than perform detailed analysis (prioritizing getting the rules back on for the majority)